authorship/data.py

```diff
@@ -36,20 +37,48 @@ def get(self):
             store_data(speakers_path, speakers_wordlist)
 
         # extract a list of frequent words
-        wordlist = [word for word, freq in Function_file.get_frequent_words(speakers_wordlist, self._n_freqwords)]
+        wordlist = [word for word, freq in get_frequent_words(speakers_wordlist, self._n_freqwords)]
 
         # build a dictionary of feature vectors
-        speakers = Function_file.filter_texts_size_new(speakers_wordlist, wordlist, self._tokens_per_sample)
+        speakers = filter_texts_size(speakers_wordlist, wordlist, self._tokens_per_sample)
 
         # convert to X, y
-        X, y = Function_file.to_vector_size(speakers)
+        X, y = to_vector_size(speakers)
 
         return X, y
 
     def __repr__(self):
         return f'data(freqwords={self._n_freqwords}; ntokens={self._tokens_per_sample})'
 
 
+class ExtractWords:
+    def __call__(self, texts):
+        yield from texts
```

Comment:
this is equivalent to:
```py
def extract_words(texts):
    yield from texts
```

Reply:
maybe okay to leave for now, because you just literally moved it from another file